integration/configs/sn1.yml

```diff
@@ -0,0 +1,47 @@
+log-config:
+  log-file:             # e.g: logs.json - by default, logs are only written to stdout
+  log-compress: true    # compress log or not
+  log-max-size-mb: 100   # maximum log file size before rotate to new file
+  log-max-age-days: 3   # maximum duration to retain old log files based on the timestamp encoded in their filename
+  log-max-backups: 10   # set the maximum number of old log files to retain
+  log-levels:           # log level of individual files 
+    common: debug
+    p2p: debug
+    metadb: info
+    dd: info
+temp-dir:
+work-dir: 
+rq-files-dir: 
+dd-service-dir: 
+
+node:
+  pastel_id: "jXYUbc59imKKoSNFdu7BcNbCBULS21MBBe8GjzfRXi9vGZrXV1y7ihJwY9ib8xTB8cSnsatvNroNJ8b7DN4Rrd"
+  pass_phrase: "passphrase"
+  server:
+    listen_addresses: "0.0.0.0"
+    port: 14444
+
+p2p:
+  listen_address: "0.0.0.0"
+  port: 14445
+  data_dir: "p2p-data"
+  bootstrap_ips: "snserver-2:14445,snserver-3:14445"
+  external_ip: "snserver-1"
+
+metadb:
+  is_leader: false
```

Comment:
should set it to `true`

Reply:
we are not using meta db - these tests only work with p2p atm